app/src/main/res/values/strings.xml

```diff
@@ -1132,6 +1132,7 @@ If any lanes are reserved for buses, please leave a note instead."</string>
     <string name="quest_leaf_type_broadleaved">Leaves</string>
     <string name="quest_leaf_type_mixed">Both are present</string>
     <string name="quest_leafType_tree_title">Does this tree have needles or leaves?</string>
+    <string name="quest_leafType_tree_is_just_a_stump">"It's just a tree stump"</string>
```

Comment:
Please use a typographic apostrophe, like in line 1111.

Reply:
done, tnx... Where have gone good old days when 640KB and ASCII was good enough for everyone :smile: